DQM/SiPixelPhase1Common/src/SiPixelCoordinates.cc

```diff
@@ -0,0 +1,916 @@
+// -*- C++ -*-
```

Comment:
If it's a compilation issue, my experience is that you need to add something to the BuildFile:
https://github.com/cms-sw/cmssw/blob/CMSSW_9_0_X/DataFormats/SiPixelDetId/BuildFile.xml
Compare what is in the DQM package that is missing in the DataFormats:
https://github.com/cms-sw/cmssw/blob/CMSSW_9_0_X/DQM/SiPixelPhase1Common/BuildFile.xml